resources/js/visual-editor/blocks/embed/embed.css

```
@@ -0,0 +1,156 @@
+.wp-block-embed.is-loading {
+    display: flex;
+    justify-content: center;
+}
+
+.wp-block-embed .wp-block-embed__placeholder-input {
+    flex: 1 1 auto;
+}
+
+/* Stops long URLs from breaking out of the no preview available screen. */
+.wp-block-embed .components-placeholder__error {
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)